src/libasr/codegen/wasm_to_x64.cpp

```diff
@@ -40,19 +41,105 @@ class X64Visitor : public WASMDecoder<X64Visitor>,
         wasm_bytes.from_pointer_n(code.data(), code.size());
     }
 
+    void visit_Return() {}
+
+    void call_imported_function(uint32_t func_idx) {
+        switch (func_idx) {
+            case 0: {  // print_i32
+                std::cerr << "Call to print_i32() will be printed as exit code\n";
+
+                // Currently, for print we are setting the value to be printed
+                // as the exit code. Later we can access/view this value from console
+                // using: echo $?
+                m_a.asm_pop_r64(X64Reg::rdi); // get exit code from stack top
+                m_a.asm_mov_r64_imm64(X64Reg::rax, 60); // sys_exit
+                m_a.asm_syscall(); // syscall
+                break;
```

Comment:
Currently, instead of printing a value, we set the value as `exit code` and can then see/view the exit code value using `echo $?`.

Reply:
Beautiful!